src/sequelize_connection_manager.ts

```diff
@@ -52,6 +52,13 @@ export class SequelizeConnectionManager {
       }
     }
 
+    config.retry = {
+      match: [
+         /SQLITE_BUSY/,
```

Comment:
```suggestion
         /SQL_BUSY/,
         /SQLITE_BUSY/,
```

The `SQL_BUSY` flag must be matched here as well.
Otherwise we get errors for the same thing in postgres, mysql or any other database that can implement a `busy` state.